### PR TITLE
removed unnecessary http headers (see http://stackoverflow.com/questions...

### PR DESCRIPTION
**Pull request by chipitsine**
I removed http response headers

X-Powered-By: ASP.NET
X-AspNet-Version: 4.0.30319
X-AspNetMvc-Version: 4.0

I hope, it will save few bytes per request on nuget.org

### CODE REVIEW

**Comment by chipitsine** [comment]
well, I just want to merge https://github.com/chipitsine/NuGetGallery/commit/619ae24a7a8ae85c870fe4bdb310eb65512e2e77 (which is signle commit, not "28 commits")

**Comment by chipitsine** [comment]
up

**Comment by chipitsine** [comment]
up

**Comment by chipitsine** [comment]
up

File: src/NuGetGallery/NuGetGallery.csproj [hunk 1]
```diff
@@ -1453,7 +1455,7 @@
           <AutoAssignPort>True</AutoAssignPort>
           <DevelopmentServerPort>80</DevelopmentServerPort>
           <DevelopmentServerVPath>/</DevelopmentServerVPath>
-          <IISUrl>http://nuget.localtest.me</IISUrl>
+          <IISUrl>http://localhost</IISUrl>
```
Comment (xavierdecoster) [inline src/NuGetGallery/NuGetGallery.csproj]: this should remain http://nuget.localtest.me

**Comment by chipitsine** [comment]
sorry, I didn't mention nuget.localtest.me again.
do not merge it, just merge headers patch

**Comment by chipitsine** [comment]
what is "cla-required" ?

**Comment by chipitsine** [comment]
what is "cla-required" ?

**Comment by yishaigalatzer** [comment]
It means you need to sign a contributor license agreement. If the change is small it is not required though.

If you clean up (rebase to dev, remove unnecessary commits) it might not be required.

The process is automated, but for some reason didn't kick in here. I will check today.

**Comment by yishaigalatzer** [comment]
@chipitsine it means that you need to sign a contributor license agreement.

The right thing here is to rebase this PR, such we have a single small commit and then we can re-evaluate (its automatic) if you need a CLA or not.

**Comment by chipitsine** [comment]
I have not figured out how to rebase it (it already looks good), so I deleted chipitsine/NugetGallery and cloned it again,

https://github.com/NuGet/NuGetGallery/pull/2489 - it's new pull request (instead of this)

Also, I signed CLA at https://cla.dotnetfoundation.org/

**Comment by xavierdecoster** [comment]
Thanks! Closing this one, and will follow up on #2489

**Pull request closed without merging**